manifests/primary/firewall.pp

```diff
@@ -5,15 +5,26 @@
 #
 class profile_kerberos_server::primary::firewall {
 
-  # This should be kadmin and kpasswd
+  # kpasswd
   ['tcp','udp'].each |$protocol| {
-
-    firewall { "212 Kerberos via ${protocol}":
+    firewall { "212 kpasswd ${protocol}":
       proto  => $protocol,
-      dport  => [
-        '464',
-        '749',
-      ],
+      dport  => '464',
+      action => 'accept',
+    }
+  }
+
+  # kadmin
+  [
+    '141.142.0.0/16',
+    '10.142.0.0/16',
+    '172.24.0.0/13',
+    '172.16.0.0/13'
```

Comment:
This works, but ideally these would be a parameter in the `profile_kerberos_server::primary::firewall` class with the array of subnets coming from hiera. That allows it to be overridden a bit easier as things change in the future, and allows the module to be used outside of NCSA.

Reply:
They're fairly urgent right now, so I'd like to merge this in as is so we can get back to running puppet on the KDC.  Otherwise I'll have to leave puppet disabled.

Then I'll open a ticket and start work on this change.  I feel like it might take a little longer because I understand nothing about hiera yet.